docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -339,12 +339,10 @@ keeping the number of segments low are:
 In many production installations which have been ingesting events for a long period of time, these suggestions alone
```

Comment:
not so much related to this PR, but the section "On the Subject of Segments" looks completely out-of-date and gives false idea of KIS always generating a lot of small segments and compaction being a mandatory requirement. would you be able to update this ?

Reply:
Do you mean we can avoid the small segments problem of KIS by setting a longer taskDuration and enabling incremental handoffs?

Reply:
segment size is not so much impacted by taskDuration but by "taskCount" and "maxRowsInSegment" settings. After https://github.com/druid-io/druid/pull/4815 , we don't create one segment per kafka partition so, in general, segments generated shouldn't necessarily be small anymore.

@pjain1 can you please do a separate PR to fix the doc based on updates in #4815 ?